sources/assets/bloodhound/customqueries.json

```diff
@@ -866,6 +858,72 @@
+        },
+        {
+            "name":"Between enabled computers (max 200)",
+            "category":"Weak ACLs",
+            "queryList":[
+                {
+                    "final":true,
+                    "query":"MATCH p=(c1:Computer {enabled: TRUE})-[:AllExtendedRights|AddMember|ForceChangePassword|GenericAll|GenericWrite|Owns|WriteDacl|WriteOwner|AddSelf|WriteSPN|AddKeyCredentialLink*1..]->(c2:Computer {enabled: TRUE}) WHERE NOT(c1.name STARTS WITH 'MSOL_') RETURN p LIMIT 200"
+                }
+            ]
+        },
+        {
+            "name": "Find computers admin to other computers",
+            "category": "Weak ACLs",
+            "queryList": [{
+                "final": true,
+                "query": "MATCH p = (c1:Computer)-[r1:AdminTo]->(c2:Computer) RETURN p UNION ALL MATCH p = (c3:Computer)-[r2:MemberOf*1..]->(g:Group)-[r3:AdminTo]->(c4:Computer) RETURN p"
+            }]
+        },
+        {
+            "name":"Between enabled users and computers (max 200)",
+            "category":"Weak ACLs",
+            "queryList":[
+                {
+                    "final":true,
+                    "query":"MATCH p=(u:User {enabled: TRUE})-[:AllExtendedRights|AddMember|ForceChangePassword|GenericAll|GenericWrite|Owns|WriteDacl|WriteOwner|AddSelf|WriteSPN|AddKeyCredentialLink*1..]->(c:Computer {enabled: TRUE}) WHERE NOT(u.name STARTS WITH 'MSOL_') RETURN p LIMIT 200"
+                }
+            ]
+        },
+        {
+            "name":"Between enabled computers and users (max 200)",
+            "category":"Weak ACLs",
+            "queryList":[
+                {
+                    "final":true,
+                    "query":"MATCH p=(c:Computer {enabled: TRUE})-[:AllExtendedRights|AddMember|ForceChangePassword|GenericAll|GenericWrite|Owns|WriteDacl|WriteOwner|AddSelf|WriteSPN|AddKeyCredentialLink*1..]->(u:User {enabled: TRUE}) WHERE NOT(u.name STARTS WITH 'MSOL_') RETURN p LIMIT 200"
+                }
+            ]
+        },
+        {
+            "name": "Objects with the AddAllowedToAct or WriteAccountRestrictions right on an enabled computer",
+            "category": "Weak ACLs",
+            "queryList": [{
+                "final": true,
+                "query": "MATCH p=(g {enabled: TRUE})-[:AddAllowedToAct|WriteAccountRestrictions]->(c:Computer {enabled: TRUE}) RETURN p"
+            }]
+        },
+        {
+            "name":"Miscellaneous direct ACLs (max 200)",
+            "category":"Weak ACLs",
+            "queryList":[
+                {
+                    "final":true,
+                    "query":"MATCH p=(u1)-[:AllExtendedRights|AddMember|ForceChangePassword|GenericAll|GenericWrite|Owns|WriteDacl|WriteOwner|AddSelf|WriteSPN|AddKeyCredentialLink*1..]->(u2) WHERE NOT(u1.name STARTS WITH 'MSOL_') AND NOT(u2.name STARTS WITH 'MSOL_') AND NOT(u1.name CONTAINS 'ADMIN') AND NOT(u2.name CONTAINS 'ADMIN') RETURN p LIMIT 200"
+                }
+            ]
+        }
```

Comment:
Imo, this section should be placed after the Kerberos delegations section, around line 220

Reply:
Done